src/nrncvode/occvode.cpp

```diff
@@ -248,12 +247,13 @@ printf("%d Cvode::init_eqn id=%d neq_v_=%d #nonvint=%d #nonvint_extra=%d nvsize=
 		int n;
 		ml = cml->ml;
 		mf = memb_func + cml->index;
-		Pfridot sc = (Pfridot)mf->ode_count;
+		nrn_ode_count_t sc = mf->ode_count;
 		if (sc && ( (n = (*sc)(cml->index)) > 0)) {
-			Pfridot s = (Pfridot)mf->ode_map;
+			nrn_ode_map_t s = mf->ode_map;
 			if (mf->hoc_mech) {
 				for (j=0; j < ml->nodecount; ++j) {
-(*s)(ieq, z.pv_ + ieq, z.pvdot_ + ieq, ml->prop[j], atv + ieq);
+//(*s)(ieq, z.pv_ + ieq, z.pvdot_ + ieq, ml->prop[j], atv + ieq);
+assert(0); // fixme
```

Comment:
this won't be executed in non-debug build. Do we have something like `nrn_assert`? Also, error message will be helpful.

Reply:
Examined the context with hoc_mech further and determined that the code fragments can be removed since all cvode related callbacks are NULL for that.